### DISCUSSION TITLE
how to use postgresql ?

### DISCUSSION THREAD

**Discussion by qualitydev515** [community/community#118950] [Programming Help]
### Body

I am going to use postgresql to use postgis.
how ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by quality01515** [comment]
Hello, i will help you again.

**Comment by qualitydev515** [comment]
Ah... Thank you

**Comment by quality01515** [comment]
No problem.
You download postgresql professional version.
And you should download postgis while installing the program.
Let me know when you have problem regarding the issue.

**Comment by qualitydev515** [comment]
Thank you really.

**Comment by quality01515** [comment]
Welcome

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
